pyre/bench/synth/generator_pep479.py

```
@@ -0,0 +1,58 @@
+
+    def check(label, fn):
+        try:
+            out.append((label, "ok", fn()))
+        except RuntimeError as e:
+            cause = type(e.__cause__).__name__ if e.__cause__ is not None else None
+            out.append((label, "runtimeerror", str(e), cause))
+        except Exception as e:  # noqa: BLE001
+            out.append((label, "other", type(e).__name__))
+
+    check("list", lambda: list(g_raise()))
+    check("for", lambda: [x for x in g_raise()])
```

Ruff (0.15.18)

[warning] Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)